debian/postinst

```diff
@@ -58,6 +60,24 @@ function ensure_paths {
     fi
 }
 
+function insert_tos_agreement {
+    touch $OS_DIR_CONFIG/config.toml
+
+    # Delete the terms part from config if it exists
+    if grep -Fxq "[terms]" $OS_DIR_CONFIG/config.toml
+    then
+        sed -i -r '/\[terms\]/,/\[/{//!d}' $OS_DIR_CONFIG/config.toml
```

Comment:
Not sure it's it's good idea to modify terms after each install

Reply:
We can debate this for sure. We could only insert the `terms` agreement if no such config variable exists.